tests/sentry/db/postgres/lookups/test_array_element_contains.py

```diff
@@ -0,0 +1,30 @@
+from unittest.mock import Mock, patch
+
+from sentry.db.postgres.lookups import ArrayElementContainsLookup
+
+
+def test_as_sql_basic_usage():
+    with (
+        patch(
+            "sentry.db.postgres.lookups.array_element_contains.ArrayElementContainsLookup.process_lhs"
+        ) as mock_process_lhs,
+        patch(
+            "sentry.db.postgres.lookups.array_element_contains.ArrayElementContainsLookup.process_rhs"
+        ) as mock_process_rhs,
```

Comment:
prefer `mock.patch.object` to `patch` -- the latter has import side-effects and is more likely to lead to leaked patches

Reply:
also it'll be way fewer characters here :)